### ISSUE TITLE
Moving to new column doesn't work in 0.9.0 version

### ISSUE THREAD

**Issue by belveryin** [sdorra/angular-dashboard-framework#107]
If I configure my dashboard layout with a widget array inside every column everything works well:

```javascript
    .config(['dashboardProvider',
        function(dashboardProvider) {
            dashboardProvider
                .structure('6-6', {
                    rows: [{
                        columns: [{
                            styleClass: 'col-md-6',
                            widgets: []
                        }, {
                            styleClass: 'col-md-6',
                            widgets: []
                        }]
                    }]
                });
        }
    ])
 ```

If I just don't configure the dashboard I can insert the widget in the first column (0.9.0 solves the issue: https://github.com/sdorra/angular-dashboard-framework/issues/73) but not move it to the second column. Then I get the error:
```
TypeError: Cannot read property 'splice' of undefined
    at angular-dashboard-framework.js:137
```
This only happens with the original layout of the dashboard. If I change to one of the predefined layouts everything works.

**Comment by albuch** [comment]
+1

**Comment by snoopyowns** [comment]
It is in function addWidgetToColumn. It is because the targetColumn.widgets is undefined since it is empty. I am trying to find a fix for it.

**Comment by snoopyowns** [comment]
In the $apply, just add:
			if (!targetColumn.widgets) {
				targetColumn.widgets = [];
			}

That should resolve it.

**Comment by sdorra** [comment]
Is already fixed in v0.10.0.
